xml/dm_registers.xml

```diff
@@ -153,8 +148,8 @@
             Debug Module after power up, including the platform's system reset
             or Debug Transport reset signals.
 
-            A debugger should pulse this bit low to ensure that the Debug
-            Module is fully reset and ready to use.
+            When first connecting, a debugger should pulse this bit low to
```

Comment:
I agree that you can switch debuggers and preserve *hart* state, but resetting the DM doesn't affect the system that's being debugged at all, right? It just affects which harts are currently selected, default system bus access size etc. These are all things that any debugger would configure anyway.

Hardware *may* require that a debugger perform a reset when first connecting. By requiring the software to do it we make everything a little more consistent.